From e81a709398fa7daa22a2d52df2b2724128d47077 Mon Sep 17 00:00:00 2001
From: Daniel Rosenberg <drosen@google.com>
Date: Mon, 2 Feb 2015 14:11:48 -0800
Subject: [PATCH 548/634] arm64: kernel: Explicitly add include in traps.c

Explicitly add #include for asm/insn.h needed for backport.
Later patches created an implicit chain, but not until v3.16

Signed-off-by: Daniel Rosenberg <drosen@google.com>
Change-Id: I68def794e767467264fa6c8b06d60cdbb9b7e9e0
---
 arch/arm64/kernel/traps.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/kernel/traps.c b/arch/arm64/kernel/traps.c
index 7ffaddd..49c667b 100644
--- a/arch/arm64/kernel/traps.c
+++ b/arch/arm64/kernel/traps.c
@@ -33,6 +33,7 @@
 
 #include <asm/atomic.h>
 #include <asm/debug-monitors.h>
+#include <asm/insn.h>
 #include <asm/traps.h>
 #include <asm/stacktrace.h>
 #include <asm/exception.h>
-- 
1.9.1

